node/core/pvf/tests/it/main.rs

```diff
@@ -58,11 +56,14 @@ impl TestHost {
 		params: ValidationParams,
 	) -> Result<ValidationResult, ValidationError> {
 		let (result_tx, result_rx) = futures::channel::oneshot::channel();
+
+		let code = sp_maybe_compressed_blob::decompress(code, 10 * 1024 * 1024).expect("Compression works");
```

Comment:
Nit: that's the original value we use

```suggestion
		let code = sp_maybe_compressed_blob::decompress(code, 16 * 1024 * 1024).expect("Compression works");
```

Reply:
This was just randomly picked by me :D